functions/http/main.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START functions_http_signed_url]
+from datetime import datetime, timedelta
+# [END functions_http_signed_url]
+
+# [START functions_http_xml]
+import json
+# [END functions_http_xml]
+
+# [START functions_http_form_data]
+import os
+import tempfile
+# [END functions_http_form_data]
+
+# [START functions_http_signed_url]
+from flask import abort
+from google.cloud import storage
+# [END functions_http_signed_url]
+
+# [START functions_http_form_data]
+from werkzeug.utils import secure_filename
```

Comment:
I believe this will work as-is, even without werkzeug in requirements.txt, but have you tested it in production just to make sure (since clearing requirements.txt)?

Reply:
Made some bugfixes and confirmed it works.